app/Http/Controllers/MobileMessageNotificationController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Notifications\CustomerMobileMessageNotification;
+use App\Repositories\CustomerRepository;
+use App\Services\MessageService;
+use Illuminate\Http\Request;
+
+class MobileMessageNotificationController extends Controller
+{
+    private $customerRepository;
+    public function __construct(CustomerRepository $customerRepository)
+    {
+        $this->customerRepository = $customerRepository;
+    }
+
+    public function store(Request $request)
+    {
+        $request->validate([
+            'customer_id' => ['required', 'integer', 'exists:customers,id'],
+            'subject' => ['required', 'string', 'max:255'],
+            'message' => ['required', 'string']
+        ]);
+        $customer = $this->customerRepository->getCustomer($request->customer_id);
+        if (!$customer) {
+            $this->sendError('Invalid customer ID supplied', 400);
+        }
+        $messageService = new MessageService();
```

Comment:
Can we use the messaging channel in the CustomerMobileMessageNotification to handle the messaging as well?
Reference `https://github.com/AltaraNg/Altara-1.0/blob/main/app/Notifications/NewOrderNotification.php#L46`